gravitee-apim-console-webui/src/management/api/history-v4/deployments-table/deployments-table.component.scss

```diff
@@ -0,0 +1,14 @@
+tr td:first-child,
```

Comment:
i's better here to use class than direct html tag
you can use `.mat-column-version` or `.mat-column-action` it's more readable. 
more info : [here](https://v16.material.angular.io/cdk/table/overview#styling-columns)

Reply:
Didn't know about that! Done